app/helpers/react_components/premium/price_option.rb

```diff
@@ -8,7 +8,8 @@ def to_s
           "premium-price-option", {
             user_signed_in: user_signed_in?,
             captcha_required: !current_user || current_user.captcha_required?,
-            recaptcha_site_key: ENV.fetch('RECAPTCHA_SITE_KEY', Exercism.secrets.recaptcha_site_key)
+            recaptcha_site_key: ENV.fetch('RECAPTCHA_SITE_KEY', Exercism.secrets.recaptcha_site_key),
+            premium_redirect_link: Exercism::Routes.premium_index_path
```

Comment:
If I test this locally, this route does not exist. `premium_path` does

Reply:
yes, it is an update by Jeremy